packages/devtools-source-editor/index.js

```diff
@@ -0,0 +1,4 @@
+import SourceEditor from "./src/source-editor";
```

Comment:
I saw console threw a error message `SyntaxError: import declarations may only appear at top level of a module`.